graveler/committed/batch_test.go

```diff
@@ -92,7 +92,7 @@ func NewFakeRangeWriter(result *committed.WriteResult, err error) *FakeRangeWrit
 }
 
 func TestBatchCloserSuccess(t *testing.T) {
-	runSuccessScenario(t)
+	runSuccessScenario(t, 500, 5)
```

Comment:
```suggestion
	runSuccessScenario(t, 502, 5)
```
Not a fan of even divisors when I can avoid them.  (I see no harm in having an even divisor, I just prefer not to have to think about whether it weakens the test.)